app/src/main/res/drawable/icon_circle_transparent.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="oval">
```

Comment:
Cirrus gives a warning that this drawable is unused. Can it be removed or will it be used later on?